libs/transactions/src/index.ts

```diff
@@ -13,3 +13,5 @@ export {
   TransactionsCommunicationService,
   TRANSACTIONS_JOURNEY_COMMUNICATION_SERIVCE,
 } from './lib/communication';
+
+export { TransactionDetailsTrackerEvent, TransactionListTrackerEvent} from './lib/model/tracker-events';
```

Comment:
I think we need an explanation of why we export it from the library. It isn't clear for a developer, because we use the events only inside the library implementation.

Reply:
Done